src/main/kotlin/no/nav/familie/ks/sak/ekstern/EksternBehandlingController.kt

```diff
@@ -24,7 +24,7 @@ import org.springframework.web.bind.annotation.RestController
 @ProtectedWithClaims(issuer = "azuread")
 class EksternBehandlingController(
```

Comment:
Hvis denne kontroller kun er for klage, kan det kalles KlageController. EksternBehandling er forvirrende siden både tilbakekreving og klage er ekstern til ks-sak

Reply:
api path kan ogå stares slik /api/klage/behandling

Reply:
Den er kun for klage, men er også ganske generisk. Den har to endepunkter for å sjekke om man kan opprette en revurdering og for å opprette revurdering. 

Synes også det var litt deilig at man skilte mellom eksterne kall og kall fra ks-frontend. Vi har allerede en klage controller som inneholder allerede endepunkter mot frontend, så da må denne logikken inn der også

Reply:
EksternBehandlingController  høres litt forvirrende ut, siden alle integrasjoner f.eks tilbakekreving, klage, pdl er egentlig ekstern til ks-sak, Disse api-ene selv om det er generisk oppretter bare klage behandling. Så jeg tror det er beste å ha en KlageController med alle api-er. Et kommentar kan legges til på api nivå for å beskrive hvem kaller api-ene

Reply:
Hvis vi vil bruke 2 controller, kan det også kalles som KlageFrontendController(eller KlageController) og KlageIntegrasjonController eller KlageEksternController